### PR TITLE
Tidy up reference style

### PR DESCRIPTION
**Pull request by mnot**
to make RFC Editor diffs smaller

### CODE REVIEW

**Review by martinthomson — COMMENTED** [review]
Can you share the tool that you used to make these changes?  This PR is impossible to review, but I would be happier reviewing the tool.  A careful description of any manual steps might suffice.

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -388,8 +391,8 @@ Therefore, when adjusting an RTT sample using peer-reported acknowledgment
 delays, an endpoint:
 
 - MAY ignore the acknowledgment delay for Initial packets, since these
-  acknowledgments are not delayed by the peer (Section 13.2.1 of
-  {{QUIC-TRANSPORT}});
+  acknowledgments are not delayed by the peer ( {{Section 13.2.1 of
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md:394]:
```suggestion
  acknowledgments are not delayed by the peer ({{Section 13.2.1 of
```
Hmm, this is clearly a change that was applied manually.

**Comment by mnot** [comment]
These are manual; if only there were a tool...

Diffs in the `.txt` are all very small, and in each case reflects correction of an existing error or slight adjustment to accommodate limitations in XML (in one case); happy to suppy (CI is broken because of tools.ietf.org).

**Comment by martinthomson** [comment]
I just uploaded this to a branch on this repository so that a diff is generated in CI (assuming that I have a little more luck in getting a build to stick).  The changes should be visible at https://quicwg.org/base-drafts/mnot-tidy-refs/ if CI ever passes.  Which of course it has not at this point...

**Comment by MikeBishop** [comment]
Seems to be working again; I fixed a lint problem, and diffs are now at https://quicwg.org/base-drafts/mnot-tidy-refs/

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -4359,7 +4363,7 @@ Endpoints could limit the content of PMTU probes to PING and PADDING frames,
 since packets that are larger than the current maximum datagram size are more
 likely to be dropped by the network.  Loss of a QUIC packet that is carried in a
 PMTU probe is therefore not a reliable indication of congestion and SHOULD NOT
-trigger a congestion control reaction; see Section 3, Bullet 7 of {{!DPLPMTUD}}.
+trigger a congestion control reaction; see {{Section 3 of DPLPMTUD}}, Bullet 7.
```
Comment (MikeBishop) [inline draft-ietf-quic-transport.md:4366]:
```suggestion
trigger a congestion control reaction; see Bullet 7 in {{Section 3 of DPLPMTUD}}.
```
This feels more natural in the output, but either is probably fine.

**Review by MikeBishop — APPROVED** [review]
Thanks for doing this, Mark.  I hadn't seen that @cabo had added this functionality to the tools, and I like that it puts the right references in the resulting HTML and XML.  I've reviewed the text diffs, and basically all the changes in output fix minor editorial things that are improvements anyway.

File: draft-ietf-quic-qpack.md [hunk 3]
```diff
@@ -644,23 +647,23 @@ In this example, Base = n - 2
...
-This string format includes optional Huffman encoding.
+The string literal defined by {{Section 5.2 of RFC7541}} is also used
+throughout. This string format includes optional Huffman encoding.
 
 HPACK defines string literals to begin on a byte boundary.  They begin with a
 single bit flag, denoted as 'H' in this document (indicating whether the string
 is Huffman-coded), followed by the Length encoded as a 7-bit prefix integer, and
 finally Length bytes of data. When Huffman encoding is enabled, the Huffman
-table from Appendix B of [RFC7541] is used without modification and Length
+table from {{Section B of RFC7541}} is used without modification and Length
```
Comment (seanturner) [inline draft-ietf-quic-qpack.md:666]: Shouldn't this be: {{Appendix B of RFC7541}}

File: draft-ietf-quic-tls.md [hunk 4]
```diff
@@ -707,7 +709,7 @@ Session resumption allows servers to link activity on the original connection
 with the resumed connection, which might be a privacy issue for clients.
 Clients can choose not to enable resumption to avoid creating this correlation.
 Clients SHOULD NOT reuse tickets as that allows entities other than the server
-to correlate connections; see Section C.4 of {{!TLS13}}.
+to correlate connections; see {{Section C.4 of TLS13}}.
```
Comment (seanturner) [inline draft-ietf-quic-tls.md:712]: {{Appendix C.4 of TLS13}}

File: draft-ietf-quic-transport.md [hunk 5]
```diff
@@ -4019,8 +4023,8 @@ acknowledged. This includes packets that are acknowledged after being declared
 lost, which can happen in the presence of network reordering. Doing so requires
 senders to retain information about packets after they are declared lost. A
 sender can discard this information after a period of time elapses that
-adequately allows for reordering, such as a PTO (Section 6.2 of
-{{QUIC-RECOVERY}}), or on other events, such as reaching a memory limit.
+adequately allows for reordering, such as a PTO ( {{Section 6.2 of
```
Comment (seanturner) [inline draft-ietf-quic-transport.md:4026]: ({{Section 6.2

File: draft-ietf-quic-transport.md [hunk 6]
```diff
@@ -4314,36 +4318,36 @@ determines that the quoted packet has actually been lost.
...
+(MPS, {{Section 4.4 of DPLPMTUD}}) for each combination of local and remote IP
 addresses.  This corresponds to the maximum datagram size.
 
 
 ### DPLPMTUD and Initial Connectivity
 
 From the perspective of DPLPMTUD, QUIC is an acknowledged Packetization Layer
-(PL). A QUIC sender can therefore enter the DPLPMTUD BASE state (Section 5.2 of
-{{!DPLPMTUD}}) when the QUIC connection handshake has been completed.
+(PL). A QUIC sender can therefore enter the DPLPMTUD BASE state ( {{Section 5.2
```
Comment (seanturner) [inline draft-ietf-quic-transport.md:4335]: ({{Section 5.2

File: draft-ietf-quic-transport.md [hunk 7]
```diff
@@ -6555,7 +6558,7 @@ considered separately.
 ### Handshake {#handshake-properties}
 
 The QUIC handshake incorporates the TLS 1.3 handshake and inherits the
-cryptographic properties described in Appendix E.1 of {{?TLS13=RFC8446}}. Many
+cryptographic properties described in Appendix E.1 of {{TLS13}}. Many
```
Comment (seanturner) [inline draft-ietf-quic-transport.md:6561]: {{Appendix E.1 of TLS13}}

**Review by ianswett — APPROVED** [review]
Thanks for doing this, recovery LGTM

File: draft-ietf-quic-qpack.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (MikeBishop) [reply draft-ietf-quic-qpack.md:666]: kramdown-rfc2629 doesn't currently recognize Appendix on the input side, but will produce "Appendix" on the output.  What Mark has is correct.

File: draft-ietf-quic-tls.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (MikeBishop) [reply draft-ietf-quic-tls.md:712]: Same.

File: draft-ietf-quic-transport.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (MikeBishop) [reply draft-ietf-quic-transport.md:6561]:
```suggestion
cryptographic properties described in {{Section E.1 of TLS13}}. Many
```

File: draft-ietf-quic-qpack.md (hunk 3 above)
Comment (mnot) [reply draft-ietf-quic-qpack.md:666]: No; kramdown-rfc2629 will do the right thing.

**Comment by mnot** [comment]
@janaiyengar I'd strongly prefer to keep this PR focused on behind-the-scenes tidying, rather than making an editorial wording choice. Beyond keeping the PR focused, doing this piecemeal is going to miss a large number of instances of "see..." because the changes represented here are only where a specific section is called out; you'd do much better to adjust that language consistently after this PR is applied (when you can do a search for "see {{ and it should consistently show all instances, whether or not there's a section being referenced).

If that's sensible to everyone, I think this is ready; tell me if I missed one of the comments.

**Comment by janaiyengar** [comment]
@mnot: Fair enough. This LGTM.

**Review by janaiyengar — APPROVED** [review]
@mart

**Comment by martinthomson** [comment]
Here are the diffs, since tools is failing so badly (I plan to work around that, but that will take some time).

https://gist.github.com/martinthomson/44906649598b56dd80c32268bb0f8a3a

I've spotted one problem there already, but it does look good so far.

File: draft-ietf-quic-transport.md (hunk 2 above)
Comment (martinthomson) [reply draft-ietf-quic-transport.md:4366]: This is the only change that I am going to take over what is here.

**Review by martinthomson — APPROVED** [review]

**Review by seanturner — APPROVED** [review]

**Comment by seanturner** [comment]
@mnot and @MikeBishop I have learned something new about kramdown-rfc2629. Cheers.

**Comment by mnot** [comment]
@afrind?

**Review by afrind — APPROVED** [review]

**Pull request merged**